### DISCUSSION TITLE
Small error in the "Woops! Rio got errors" screen

### DISCUSSION THREAD

**Discussion by celebrith** [raphamorim/rio#658] [General]
I wasn't quite sure how to let you know about this.  The URL at the bottom of the "Woops! Rio got errors" screen doesn't work.
https://raphamorim.io/rio/docs/#configuration-file gives a page not found error.
It looks like it should be https://raphamorim.io/rio/docs/config instead (it's easy enough to go to the github page and click Config at the top menu).  Super minor issue.

**Comment by celebrith** [comment]
This came up because the 0.1.12 version didn't like the following line in the config.toml, which has worked in the previous (and I thought it was the default, but I'm not positive):
cursor = '▇'

I replaced it with the default cursor struct from the docs, and it's working fine again:
[cursor]
shape = 'block'
blinking = false

**Reply by raphamorim** [reply]
Yes! that's correct 👌 
It was a breaking change introduced on 0.1.12 🙏 

Good call with the URL will fix right away.

**Reply by celebrith** [reply]
I did appreciate the graceful handling - hit enter twice, and it loads a default config.  It was a great way to handle it (and presumably if someone were to make a typo in making changes to the config, etc.).
